pkg/awsds/sessions_test.go

```
@@ -61,8 +61,8 @@ func TestNewSession_AssumeRole(t *testing.T) {
 		settings := AWSDatasourceSettings{
 			AssumeRoleARN: roleARN,
 		}
-		os.Setenv(AllowedAuthProvidersEnvVarKeyName, "default")
-		os.Setenv(AssumeRoleEnabledEnvVarKeyName, "true")
+		require.NoError(t, os.Setenv(AllowedAuthProvidersEnvVarKeyName, "default"))
```

just curious what this does, is it checking that setenv doesn't error out? seems maybe overly cautious no? Or is there an expected error that could happen?

---

I think it's unlikely an error will happen, but it would be nice to know if it does instead of silently continuing in any case.